greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-6308: Release EN Form WYSIWYG block

Ref: https://jira.greenpeace.org/browse/PLANET-6308
Cf. https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/453

**Release of ENForm WYSIWYG block**
![Screenshot from 2022-01-11 10-11-56](https://user-images.githubusercontent.com/617346/148913992-86d730df-5bd2-4b8a-a644-b5840c3b584d.png)

- Deprecation of previous version
- Fixes for existing blocks
- Removal of old code

## Test

Release is available on https://www-dev.greenpeace.org/test-saturn/
- Check that it doesn't break existing ENForm blocks (find existing blocks in Blocks > Usage)
- Check that we can still add this block to a page and select various options
